### ISSUE TITLE
Make feature stroke width configureable

### ISSUE THREAD

**Issue by ygerlach** [orchestracities/map-panel#95]
Having many polygons next to each other makes the stroke overlap. When zoomed out, only the top rendered polygons appear. That makes it look like only the top rendered polygon is there.
One way to fix this would be to make the stroke width of polygons configureable. So you would be able to set it to zero.
Another way might be to make the stroke width smaller when the view is zoomed out.

Screenshots:
Zoomed in area:
![image](https://user-images.githubusercontent.com/100762533/220915173-3aeb10ca-5eea-4ff7-8969-425a32006237.png)
Same area, zoomed out: (the green polygons "disappeared")
![image](https://user-images.githubusercontent.com/100762533/220915407-d66d28bf-4fc5-4481-84db-1ca7461a9c68.png)

**Comment by nathanoday** [comment]
Yes, please! It's really difficult to see closely aligned polygons because of the stroke width. Compared with the stock geomap panel, it's borderline unusable in many cases.

**Comment by bobleijnse** [comment]
I'm having the same problem. Would there be a temporary (hard-coding) fix to solve this?

**Comment by mrbesen** [comment]
I have no idea of creating grafana plugins and react applications.

But this looks like a prommising line:

https://github.com/orchestracities/map-panel/blob/master/src/layers/data/markersLayer.tsx#L390

**Comment by ygerlach** [comment]
> I'm having the same problem. Would there be a temporary (hard-coding) fix to solve this?

For me removing the stroke or hard-coding its width to zero should resolve my problem.

**Comment by ygerlach** [comment]
Looks like this is related to #76
